rascal-textmate-core/src/main/rascal/lang/oniguruma/Conversion.rsc

```diff
@@ -158,7 +158,9 @@ default RegExp toRegExp(Grammar _, Condition c) {
     Converts a character range to a regular expression.
 }
 
-RegExp toRegExp(Grammar _, range(begin, end))
+RegExp toRegExp(Grammar _, range(char, char))
+    = regExp("<encode(char)>", []);
+default RegExp toRegExp(Grammar _, range(begin, end))
```

Comment:
These changes are unrelated to the goal of this PR and shouldn't be here, but they are...

Reply:
tss ;-)